src/hooks/journals/shared.ts

```diff
@@ -55,7 +55,7 @@ export async function loadDocuments({
     maxBytes,
     offsets,
 }: LoadDocumentsProps): Promise<LoadDocumentsResponse> {
-    if (!client || !journalName) {
+    if (!client || !journalName || journalName.length === 0) {
```

Comment:
We now will return an empty string if we cannot figure out the `kind` of a journal. So making sure there is _some_ string here.